src/benchmark_environments/testing/envs.py

```diff
@@ -211,11 +211,9 @@ def step(self, action):
         """Step method for CountingEnv."""
         if self.timestep is None:
             raise RuntimeError("Need to reset before first step().")
-        if self.timestep > self.episode_length:
```

Comment:
Why are we removing this? I don't think we call `step()` on `CountingEnv` after `done` is True?

Reply:
Removed this error so I can testing `AbsorbAfterDone` wrapper, which needs environment to continue progressing after the first `done=True`.

Reply:
Hmmmm that sounds right actually

Reply:
Fixed, thanks for catching this